common/common-rest/src/test/java/org/apache/servicecomb/common/rest/TestAbstractRestInvocation.java

```diff
@@ -440,6 +440,26 @@ protected void sendResponse(Response response) {
     // just log, check nothing
   }
 
+  @Test
+  public void sendResponseQuietlyExceptionOnNullInvocation(@Mocked Response response) {
+    restInvocation = new AbstractRestInvocationForTest() {
+      @Override
+      protected void doInvoke() {
+      }
+
+      @Override
+      protected void sendResponse(Response response) {
+        throw new Error("");
+      }
+    };
+    initRestInvocation();
+    restInvocation.invocation = null;
+
+    restInvocation.sendResponseQuietly(response);
+
+    // just log, check nothing, and should not throw NPE
```

Comment:
Can we add an integration test for this scenario? Call a not exists service and got a 404.

Reply:
Done, please review